src/ctelnet.cpp

```diff
@@ -2011,7 +2011,7 @@ void cTelnet::setGMCPVariables(const QByteArray& msg)
         mpHost->processDiscordGMCP(packageMessage, data);
     }
 
-    if (mpHost->mAcceptServerMedia && packageMessage.startsWith(QStringLiteral("Client.Media").toLower())) {
+    if (mpHost->mAcceptServerMedia && packageMessage.toLower().startsWith(QStringLiteral("client.media"))) {
```

Comment:
ℹ️ The `QString::toLower()` is redundant here, as the `QString::startsWith()` will take a second optional argument to render it non-case sensitive (NOT the default) of the form `Qt::CaseInsensitive`!

Reply:
@SlySven are you reviewing the line I deleted? :)

Reply:
The new one has "toLower().startsWith()" from what I see

Reply:
Would someone like me to consider adding case insensitivity to the other code in that block in another PR?  Client.GUI, Client.Map, the Discord stuff could use small revisions.

Reply:
Ahh, that would be great. It's hard to see them from the GitHub view.

Reply:
Sorry to be late getting back to you but the point I was trying to make was the extra `QString` construction done because of the use of `toLower()` is not needed if you just add the `Qt::CaseInsensitve` option to the original `startsWith(...)` - FTR this is something that CLAZY will pick up and moan about - see item 30 here: https://www.kdab.com/uncovering-32-qt-best-practices-compile-time-clazy/ !